SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -205,7 +209,7 @@ void CTurbSSTSolver::Postprocessing(CGeometry *geometry, CSolver **solver_contai
   }
 
   AD::StartNoSharedReading();
-
+  // nijso asks: why do we do this?
   auto* flowNodes = su2staticcast_p<CFlowVariable*>(solver_container[FLOW_SOL]->GetNodes());
```

Comment:
Because it allows de-virtualizing the access to the flow primitives and gradients.
```suggestion
  auto* flowNodes = su2staticcast_p<CFlowVariable*>(solver_container[FLOW_SOL]->GetNodes());
```